src/app.ts

```diff
@@ -1,60 +1,24 @@
 import express, {Application, NextFunction, Request, Response} from "express";
-import axios, {AxiosResponse} from "axios";
+import { logToDB } from "./middleware/logToDB";
+
 const app: Application = express();
 const port: number = 3000;
 
+const usersRoutes = require('./routes/users');
```

Comment:
`require` isn't used in Typescript. Use `import` instead.

Reply:
Done